react-app/src/components/UploadManager.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { 
+  TextField,
+  IconButton,
+  Typography
+} from '@material-ui/core'
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward'
+
+class UploadManager extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      name: ''
+    }
+
+    this.onInputChange = this.onInputChange.bind(this)
+  }
+
+  onInputChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onUploadClick = () => {
+    localStorage[this.state.name] = this.props.photo.dataUri
```

Comment:
I've noticed that local storage names persist across usages. Not a huge deal, and honestly a possible feature, but it could overload the local storage if we're not careful. I think the long term would be to create a separate UI for managing loaded faces, but for now I think it would be best to keep each load of the app "fresh"

I'm not super familiar with how `localStorage` works, but could we set it to clear when the app is reloaded? This could probably be something in a `useEffect()` method in `App.jsx`

Reply:
There's actually a separate storage mechanism called `sessionStorage` that, to my knowledge, works exactly like `localStorage` but is wiped every time the page is reloaded. Should be an easy switch over to that.